utils/build-command-index.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Viktor Söderqvist <viktor.soderqvist@est.tech>
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import os
+import re
+import json
+import argparse
+import sys
+import yaml
+from datetime import date
+
+import command_syntax
+
+parser = argparse.ArgumentParser(prog="build-command-index.py",
+                                 description='Create an overview page of all commands, commands/index.md')
+parser.add_argument('--suffix', default='.md',
+                    help='Suffix to use in internal links instead of .md for non-manpage usage')
+parser.add_argument('--man', action='store_true', help='Generate markdown for man page')
+parser.add_argument('--groups-json', help='groups.json')
+parser.add_argument('--commands-per-group-json', help='commands-per-groups.json')
+args = parser.parse_args()
+
+with open(args.groups_json, "r") as f:
+    groups = json.load(f)
+with open(args.commands_per_group_json, "r") as f:
+    commands_per_group = json.load(f)
+
+if args.man:
+    print("---")
+    print("title: VALKEY-COMMANDS(7)")
+    print('header: Valkey Command Manual')
+    print('footer: Valkey Documentation')
+    #print('date: %s' % self.date)
```

Comment:
```suggestion
```

Reply:
I was considering adding a date, but obviously I didn't finish this. Perhaps the timestamp of the file `args.commands_per_group_json` could be used.